src/aof.c

```diff
@@ -1904,34 +1919,6 @@ void backgroundRewriteDoneHandler(int exitcode, int bysignal) {
          * guarantee atomicity for this switch has already happened by then, so
          * we don't care what the outcome or duration of that close operation
          * is, as long as the file descriptor is released again. */
-        if (server.aof_fd == -1) {
```

Comment:
the whole thing got removed in https://github.com/redis/redis/pull/9788/files#diff-ade042c73ad8dc4e34f1a75b2a9e26a5342d1dee539a6188c2565feea0372a20R717